images/base-python-adapter/swagger_server/server_logging.py

```diff
@@ -0,0 +1,86 @@
+#  Copyright 2023 VMware, Inc.
+#  SPDX-License-Identifier: Apache-2.0
+import logging
+import os
+from configparser import ConfigParser
+from logging.handlers import RotatingFileHandler
+from typing import Optional
+
+log_handler: Optional[RotatingFileHandler] = None
+
+
+def setup_logging(
+    filename: str, file_count: int = 5, max_size: int = 10_485_760
+) -> None:
+    """
+    :param filename The name of the file to log to
+    :param file_count The total number of files to retain. Defaults to 5.
+    :param max_size The maximum size in bytes of each file before the file
+                automatically rotates to a new one. Defaults to '10_485_760' (10 MiB).
+    """
+    try:
+        global log_handler
+        log_handler = RotatingFileHandler(
+            os.path.join(os.sep, "var", "log", filename),
+            maxBytes=max_size,
```

Comment:
looks like we ended up going with maxBytes instead of time-based or linecount-based. any reason why?